brigadier/worker/app.py

```diff
@@ -18,14 +18,10 @@
     broker_url = f'pyamqp://{RABBIT_USER}:{RABBIT_PASS}@' \
         f'{RABBIT_HOST}:{RABBIT_PORT}/{RABBIT_VHOST}'
 else:
-    broker_url = None
+    broker_url = None  # pragma: no cover
```

Comment:
why are you disabling this line from coverage count?
do you consider, there's no possible way to test it?